tests/integration/explain/default/top_with_count_test.go

```diff
@@ -124,8 +124,8 @@ func TestDefaultExplainTopLevelCountRequestWithFilter(t *testing.T) {
 							},
 							"spans": []dataMap{
 								{
-									"start": "/3",
-									"end":   "/4",
+									"start": "/\x8b",
+									"end":   "/\x8c",
```

Comment:
though: this is pain. Would be nice to have it extracted so that if it changes again, we don't need to go through it again.

Reply:
Explain was actually pretty easy considering how invasive they are by nature, the unit tests were much more of a pain.

This might get changed slightly before merge anyway - the http client does not like this encoding and it caught me out.

Reply:
This has been reverted, the encoding was not json friendly and needed to be prettied up in order to work with the http client.